src/redux/action.js

```diff
@@ -7,7 +7,7 @@ const GET_CART = 'GET_CART';
 const ADD_CART = 'ADD_CART';
 const DELETE_CART = 'DELETE_CART';
 const SET_AUTH = 'SET_AUTH';
-const GET_ORDERS = 'GET_ORDERS';
+const GET_ORDERS = 'GET_ORDER';
```

Comment:
I think we changed this on Monday. Went with 'GET_ORDERS' since it's grabbing all orders instead of one single order